Tests/WorkspaceTests/SourceControlPackageContainerTests.swift

```diff
@@ -328,7 +328,7 @@ class SourceControlPackageContainerTests: XCTestCase {
     func testPreReleaseVersions() throws {
         let fs = InMemoryFileSystem()
 
-        let repoPath = AbsolutePath.root
+        let repoPath = AbsolutePath.root.appending("SourceCache")
```

Comment:
I'm not sure I can connect the description of the PR to the actual change here. It mentions relative path, but only an absolute is stored here. How does adding a `SourceCache` component make this better for Windows?

Reply:
Please add this as a comment in code so that it's visible when reviewed it in the future by someone else and not "cleaned up" by accident.

Reply:
Thanks!